kvmagent/kvmagent/test/ha_testsuite/test_ceph_fencer.py

```
@@ -0,0 +1,111 @@
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)